editor/export/project_export.cpp

```diff
@@ -1276,6 +1276,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	server_strip_message = memnew(Label);
 	server_strip_message->set_visible(false);
 	server_strip_message->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
+	server_strip_message->set_size(Size2(500, 1) * EDSCALE);
```

Comment:
We should use `set_custom_minimum_size` and force the minimum size, not set the default size, this is unreliable.

Reply:
One characteristic of this series of issues is that during first `Label::_shape()`, the size assigned to the label by the container has not been updated, causing the label's initial width to be 0, so a relatively large minimum height is calculated. 

https://github.com/godotengine/godot/blob/26b1fd0d842fa3c2f090ead47e8ea7cd2d6515e1/scene/gui/label.cpp#L104-L106

If the initial width is large enough, this won't happen. The minimum size is calculated as a single line height.

Reply:
Enforcing a minimum size is still more reliable because it doesn't depend on the order of events. You just need to pick a size which will be within the dialog's own minimum size, so there is no visible difference for the end user.